src/Sentry.Maui.CommunityToolkit.Mvvm/CtMvvmMauiElementEventBinder.cs

```diff
@@ -0,0 +1,157 @@
+using System.Windows.Input;
+using CommunityToolkit.Mvvm.Input;
+
+namespace Sentry.Maui.CommunityToolkitMvvm;
+
+/// <summary>
+/// Scans all elements for known commands that are implement
+/// </summary>
+public class CtMvvmMauiElementEventBinder(IHub hub) : IMauiElementEventBinder
+{
+    /// <summary>
+    /// Binds to the element
+    /// </summary>
+    /// <param name="element"></param>
+    /// <param name="addBreadcrumb"></param>
+    public void Bind(VisualElement element, Action<BreadcrumbEvent> addBreadcrumb) => Iterate(element, true);
+
+    /// <summary>
+    /// Unbinds from the element
+    /// </summary>
+    /// <param name="element"></param>
+    /// <exception cref="NotImplementedException"></exception>
+    public void UnBind(VisualElement element) => Iterate(element, false);
+
+
+    private void Iterate(VisualElement element, bool bind)
+    {
+        switch (element)
```

Comment:
Hm, is this switch statement likely to grow over time? It seems to essentially map from a visual element to a command. Potentially we could store this default map in some options and that way people could easily add custom mappings. I'm not sure if that's necessary or over engineering for a problem that doesn't exist though.

Reply:
I don't want to overcomplicate this right now.  There is unlikely to be a lot of "out of the box" command bindings in the next few major MAUI releases.